app/services/exactAlarmService.ts

```diff
@@ -0,0 +1,13 @@
+import { NativeModules } from 'react-native';
+
+const { ExactAlarmModule } = NativeModules;
+
+export const requestExactAlarmPermission = async () => {
+  try {
+    const granted = await ExactAlarmModule.requestExactAlarmPermission();
+    return granted; // true if permission is already granted
+  } catch (error) {
+    console.log('Error requesting exact alarm permission:', error);
+    return false;
+  }
+};
```

Comment:
Let's add some comments to document the method:
```

export const requestExactAlarmPermission = async (): Promise<boolean> => {
  try {
    // Request permission from the native module
    const permissionGranted = await ExactAlarmModule.requestExactAlarmPermission();
    
    // Return true if permission is granted, otherwise false
    return permissionGranted;
  } catch (error) {
    console.error('Error requesting exact alarm permission:', error.message || error);    
    // Return false to indicate failure in obtaining permission
    return false;
  }
};
```